utils/rowgroup/rowgroup.h

```diff
@@ -1560,11 +1561,11 @@ class RowGroup : public messageqcpp::Serializeable
   uint32_t sTableThreshold = 20;
   std::shared_ptr<bool[]> forceInline;
 
-  static const uint32_t headerSize = 18;
-  static const uint32_t rowCountOffset = 0;
-  static const uint32_t baseRidOffset = 4;
-  static const uint32_t statusOffset = 12;
-  static const uint32_t dbRootOffset = 14;
+  static const uint64_t headerSize = 18;
```

Comment:
The `getHeaderSize()` signature should also be fixed (https://github.com/mariadb-corporation/mariadb-columnstore-engine/pull/3289/files#diff-f3dffeb8ad50140337201f2ed7ecd27d3ecc86aa2a9b714cdeedfb82dc454edcR1466)